homeassistant/components/lg_soundbar/media_player.py

```diff
@@ -104,7 +107,8 @@ def handle_event(self, response):
                 self._equaliser = data["i_curr_eq"]
             if "s_user_name" in data:
                 self._name = data["s_user_name"]
-        self.schedule_update_ha_state()
+        if self.hass is not None:
```

Comment:
The event callback should be registered in entity coroutine method `async_added_to_hass`.

Reply:
Ah, that makes sense.

Reply:
We can remove this check now.